### PR TITLE
[PM-13376] Phase 2 - SDK split

### PR DESCRIPTION
**Pull request by Hinton**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-13376

## 📔 Objective

Phase 2 of the SDK repository split.

SDK-internal is a new repository split from the latest main of https://github.com/bitwarden/sdk. It removes all secrets manager language bindings and binaries.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: .github/workflows/rust-test.yml [hunk 1]
```diff
@@ -46,7 +45,7 @@ jobs:
 
       - name: Test
         # Termporarily exclude NAPI due to a test error on windows
```
Comment (dani-garcia) [inline .github/workflows/rust-test.yml]: We should remove this comment now that it's not applicable

File: .github/workflows/version-bump.yml [hunk 2]
```diff
@@ -89,73 +80,11 @@ jobs:
...
-          npm version ${{ inputs.version_number }}
-
-      - name: Bump napi crate Version
-        if: ${{ inputs.project == 'napi' }}
-        run: cargo set-version -p bitwarden-napi ${{ inputs.version_number }}
-
       ### bitwarden
       - name: Bump bitwarden crate Version
         if: ${{ inputs.project == 'bitwarden' }}
         run: cargo set-version -p bitwarden ${{ inputs.version_number }}
```
Comment (dani-garcia) [inline .github/workflows/version-bump.yml]: We don't have a `bitwarden` crate anymore do we?

Comment (Hinton) [reply .github/workflows/version-bump.yml]: We don't fixed.

File: .github/workflows/version-bump.yml [hunk 3]
```diff
@@ -89,72 +80,10 @@ jobs:
...
-          npm ci
-          npm version ${{ inputs.version_number }}
-
-      - name: Bump napi crate Version
-        if: ${{ inputs.project == 'napi' }}
-        run: cargo set-version -p bitwarden-napi ${{ inputs.version_number }}
-
       ### bitwarden
       - name: Bump bitwarden crate Version
         if: ${{ inputs.project == 'bitwarden' }}
```
Comment (dani-garcia) [inline .github/workflows/version-bump.yml]: I think we should also update the input and name to match:
```suggestion
      - name: Bump bitwarden-core crate Version
        if: ${{ inputs.project == 'bitwarden-core' }}
```

File: .github/workflows/version-bump.yml [hunk 4]
```diff
@@ -10,15 +10,6 @@ on:
         type: choice
         options:
           - bitwarden
```
Comment (dani-garcia) [inline .github/workflows/version-bump.yml]:
```suggestion
          - bitwarden-core
```

**Review by dani-garcia — COMMENTED** [review]
Just some missing names in the workflow, otherwise LGTM

File: .github/workflows/version-bump.yml (hunk 3 above)
Comment (Hinton) [reply .github/workflows/version-bump.yml]: True.

**Review by dani-garcia — APPROVED** [review]

**Pull request merged**